static/resources/yaml/observability_pipelines/datadog/aws_eks_rc.yaml

```diff
@@ -0,0 +1,76 @@
+datadog:
+  apiKey: "<datadog_api_key>"
+  pipelineId: "<observability_pipelines_configuration_id>"
+  site: "datadoghq.com"
+
+## The remote configuration env var must be set so that the OP worker 
+## boots up with RC enabled. In this Helm chart, the pipelineConfig 
+## key is notably omitted in lieu of retrieving the config from Datadog.
+env:
```

Comment:
We don't need this env var- you can set `datadog.remoteConfigurationEnabled: true` instead, which is the officially supported way.